packages/core/src/routes/account/mfa-verifications.ts

```diff
@@ -20,6 +26,37 @@ export default function mfaVerificationsRoutes<T extends UserRouter>(
     signInExperiences: { findDefaultSignInExperience },
   } = queries;
 
+  router.get(
+    `${accountApiPrefix}/mfa-verifications`,
+    koaGuard({
+      response: userMfaVerificationResponseGuard,
+      status: [200, 400, 401],
+    }),
+    async (ctx, next) => {
+      const { id: userId, scopes } = ctx.auth;
+      const { fields } = ctx.accountCenter;
+
+      // If MFA field is not enabled, return empty array
+      if (
+        fields.mfa !== AccountCenterControlValue.Edit &&
+        fields.mfa !== AccountCenterControlValue.ReadOnly
+      ) {
+        ctx.status = 200;
```

Comment:
Should we throw 401 instead? 200 with an empty response can be misleading.

Reply:
updated